restapi/src/main/java/io/stargate/web/docsapi/resources/CollectionsResource.java

```diff
@@ -0,0 +1,232 @@
+package io.stargate.web.docsapi.resources;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.db.datastore.DataStore;
+import io.stargate.db.schema.Table;
+import io.stargate.web.docsapi.dao.DocumentDB;
+import io.stargate.web.docsapi.models.DocCollection;
+import io.stargate.web.docsapi.service.CollectionService;
+import io.stargate.web.models.Error;
+import io.stargate.web.models.ResponseWrapper;
+import io.stargate.web.resources.Converters;
+import io.stargate.web.resources.Db;
+import io.stargate.web.resources.RequestHandler;
+import io.swagger.annotations.*;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api(
+    produces = MediaType.APPLICATION_JSON,
+    consumes = MediaType.APPLICATION_JSON,
+    tags = {"documents"})
+@Path("/v2/namespaces/{namespace-id: [a-zA-Z_0-9]+}")
+@Produces(MediaType.APPLICATION_JSON)
+public class CollectionsResource {
+  @Inject private Db db;
+  private static final ObjectMapper mapper = new ObjectMapper();
+  private static final CollectionService collectionService = new CollectionService();
+
+  @GET
+  @ApiOperation(value = "List collections in namespace")
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 200, message = "OK", response = ResponseWrapper.class),
+        @ApiResponse(code = 401, message = "Unauthorized"),
+        @ApiResponse(code = 500, message = "Internal server error", response = Error.class)
+      })
+  @Path("collections")
+  @Consumes("application/json")
+  @Produces("application/json")
+  public Response getCollections(
+      @ApiParam(
+              value =
+                  "The token returned from the authorization endpoint. Use this token in each request.",
```

Comment:
Outside of this PR but we should probably have some common swagger strings somewhere.